gitops/overlays/training/patches/deployments.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     spec:
       containers:
         - name: canada-dental-care-plan-frontend
-          image: dtsrhpdevscedacr.azurecr.io/canada-dental-care-plan/canada-dental-care-plan-frontend:0.0.0-36575c2d-00507
+          image: dtsrhpdevscedacr.azurecr.io/canada-dental-care-plan/canada-dental-care-plan-frontend:0.0.0-dee3652b-00520
```

Comment:
`staging` cannot find that image tag... I looke at TC and it  should be `0.0.0-57aeeae1-00520`. Fix it for training and staging.

Reply:
done